aea/cli_gui/__init__.py

```diff
@@ -195,24 +199,10 @@ def search_registered_items(item_type: str, search_term: str):
 
 def create_agent(agent_id: str):
     """Create a new AEA project."""
-    if (
-        _call_aea(
-            [
-                sys.executable,
-                "-m",
-                "aea.cli",
-                "create",
-                "--local",
-                agent_id,
-                "--author",
-                DEFAULT_AUTHOR,
-            ],
-            app_context.agents_dir,
-        )
-        == 0
-    ):
-        return agent_id, 201  # 201 (Created)
-    else:
+    ctx = Context(cwd=app_context.agents_dir)
+    try:
+        cli_create_aea(ctx, agent_id, DEFAULT_AUTHOR, local=True, empty=False)
```

Comment:
We should configure the gui so it can only be run once the user has run `aea init`. This way we have the author set, rather than having to use the default author.

Reply:
Let's do it in a separate PR. I agree with proposed approach thou.